internal/website/content/tutorials/order.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Order"
+date: 2019-07-09T10:22:39-04:00
+draft: true
+showInSidenav: false  # only for sections (any level)
```

Comment:
Why these? Seems like it should be similar to guestbook

Reply:
I don't know what they mean; `hugo new` inserted them, I think. Removed.